### PR TITLE
Lon 691

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: LongoMatch.GUI/Gui/TreeView/CategoriesFilterTreeView.cs [hunk 1]
```diff
@@ -28,69 +28,88 @@ namespace LongoMatch.Gui.Component
...
 	{
+		Project project;
+		EventsFilter filter;
 
-		public CategoriesFilterTreeView () : base ()
+		public CategoriesFilterTreeView ()
 		{
-			firstColumnName = Catalog.GetString ("Category");
-			HeadersVisible = false;
 		}
```
Comment (vmartos) [inline LongoMatch.GUI/Gui/TreeView/CategoriesFilterTreeView.cs]: Empty constructor

File: LongoMatch.GUI/Gui/TreeView/PlayersFilterTreeView.cs [hunk 2]
```diff
@@ -32,6 +32,8 @@ public class PlayersFilterTreeView: FilterTreeViewBase
 		Team local, visitor;
 		Player localTeam, visitorTeam;
 		TreeIter localIter, visitorIter;
+		Project project;
```
Comment (vmartos) [inline LongoMatch.GUI/Gui/TreeView/PlayersFilterTreeView.cs]: Project is never used.
in 
`public void SetFilter (EventsFilter filter, Project project)
		{
			this.local = project.LocalTeamTemplate;
			this.visitor = project.VisitorTeamTemplate;
			localTeam.Name = local.TeamName;
			visitorTeam.Name = visitor.TeamName;
			this.project = project;
			this.filter = filter;
			FillTree ();
		}`
you just use Local team and vistor team in the PlayersFilterTreeView class, but project is not necessary.

File: LongoMatch.GUI/Gui/TreeView/CellRendererButton.cs [hunk 3]
```diff
@@ -68,15 +66,15 @@ public override void GetSize (Widget widget, ref Rectangle cell_area, out int x_
 			IDrawingToolkit tk = Config.DrawingToolkit;
 
 			using (IContext context = new CairoContext (window)) {
-				Point pos = new Point (cellArea.X, cellArea.Y + 2);
-				int width = cellArea.Width;
+				int width = cellArea.Width - StyleConf.FilterTreeViewOnlyOffset;
 				int height = cellArea.Height - 4;
+				Point pos = new Point (cellArea.X + backgroundArea.Width - cellArea.Width, cellArea.Y + 2);
```
Comment (vmartos) [inline LongoMatch.GUI/Gui/TreeView/CellRendererButton.cs]: What's the meaning of 2?
Maybe a variable is needed to know the meaning as in StyleConf.FilterTreeViewOnlyOffset

Comment (ylatuya) [inline LongoMatch.GUI/Gui/TreeView/CellRendererButton.cs]: It's the top/bottom offset. It needs another constants like FilterTreeViewOnlyOffset

**Comment by vmartos** [comment]
:+1:

**Pull request merged**